### PR TITLE
Beta Release 0.6.0

### PR DESCRIPTION
**Pull request by DPoplin**
This PR updates the Server SDK to 0.6.0 after changes to word separation and API descriptions.

Also added basic a CHANGELOG.md

### CODE REVIEW

File: PaypalServerSdk.Standard/Models/Order.cs [hunk 1]
```diff
@@ -120,7 +120,7 @@ public Order(
...
+        /// An array of request-related HATEOAS links. To complete payer approval, use the `approve` link to redirect the payer. The API caller has 6 hours (default setting, this which can be changed by your account manager to 24/48/72 hours to accommodate your use case) from the time the order is created, to redirect your payer. Once redirected, the API caller has 6 hours for the payer to approve the order and either authorize or capture the order. If you are not using the PayPal JavaScript SDK to initiate PayPal Checkout (in context) ensure that you include `application_context.return_url` is specified or you will get "We're sorry, Things don't appear to be working at the moment" after the payer approves the payment.
```
Comment (wickerpopstar) [inline PaypalServerSdk.Standard/Models/Order.cs:123]: This is kind of an interesting change...wouldn't this only change if the schema was updated?

File: PaypalServerSdk.Standard/Models/ShipmentCarrier.cs [hunk 2]
```diff
@@ -1592,10 +1592,10 @@ public enum ShipmentCarrier
         YURTICIKARGO,
 
         /// <summary>
-        /// CNPAYPALPACKAGE.
+        /// CNPaypalPACKAGE.
```
Comment (wickerpopstar) [inline PaypalServerSdk.Standard/Models/ShipmentCarrier.cs:1595]: This seems like it's breaking convention. I wouldn't expect the word ignore to apply to enums I guess.

File: PaypalServerSdk.Standard/Models/Order.cs (hunk 1 above)
Comment (DPoplin) [reply PaypalServerSdk.Standard/Models/Order.cs:123]: Yeah this is a weird one. Makes me think maybe something in the schema was updated and we didn't catch it with the last deploy? Not sure.

File: PaypalServerSdk.Standard/Models/ShipmentCarrier.cs (hunk 2 above)
Comment (DPoplin) [reply PaypalServerSdk.Standard/Models/ShipmentCarrier.cs:1595]: Oof yeah. I'll bring this one to APIMatic

File: doc/models/paypal-experience-landing-page.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 
-# Pay Pal Experience Landing Page
+# Paypal Experience Landing Page
```
Comment (wickerpopstar) [inline doc/models/paypal-experience-landing-page.md:2]: All of these titles are a little funny now. I get that it's just matching the programming model name, but in that case, I don't even know why there are spaces.

Comment (wickerpopstar) [reply doc/models/paypal-experience-landing-page.md:2]: TBF, they were funny before with the spaces, too, so it's an improvement!

**Pull request closed without merging**